user/api.go

```diff
@@ -171,16 +173,29 @@ func (a *Api) GetUsers(res http.ResponseWriter, req *http.Request) {
 	} else if !tokenData.IsServer {
 		a.sendError(res, http.StatusUnauthorized, STATUS_UNAUTHORIZED)
 
+	} else if len(req.URL.Query()) == 0 {
+		a.sendError(res, http.StatusBadRequest, STATUS_NO_QUERY)
+
 	} else if role := req.URL.Query().Get("role"); role != "" && !IsValidRole(role) {
 		a.sendError(res, http.StatusBadRequest, STATUS_INVALID_ROLE)
 
-	} else if role == "" {
-		a.sendError(res, http.StatusBadRequest, STATUS_NO_QUERY)
-
-	} else if users, err := a.Store.FindUsersByRole(role); err != nil {
-		a.sendError(res, http.StatusInternalServerError, STATUS_ERR_FINDING_USR, err.Error())
+	} else if userIds := strings.Split(req.URL.Query().Get("id"), ","); len(userIds[0]) > 0 && role != "" {
```

Comment:
It can't be null. `URL.Query().Get("id")` will return an empty string if either `id` doesn't exist, or has been passed with `?id=`. What will happen is that it will fall through to line 197, where it will throw `STATUS_PARAMETER_UNKNOWN`.